src/Workspaces/MSBuildTest/MSBuildWorkspaceTests.cs

```diff
@@ -3043,76 +2803,30 @@ public void DisposeMSBuildWorkspaceAndServicesCollected()
 
             // MSBuildWorkspace doesn't have a cache service
             Assert.Null(workspace.UseReference(static w => w.CurrentSolution.Services.CacheService));
+            // update root blindly again, after observing encoding, see that now encoding is known
+            var doc3 = document.WithSyntaxRoot(gen.CompilationUnit()); // empty CU
+            var doc3text = await doc3.GetTextAsync();
+            Assert.NotNull(doc3text.Encoding);
+            Assert.Equal(encoding.EncodingName, doc3text.Encoding.EncodingName);
+            var doc3tree = await doc3.GetSyntaxTreeAsync();
+            Assert.Equal(doc3text.Encoding, doc3tree.GetText().Encoding);
+            Assert.Equal(doc3text.Encoding, doc3tree.Encoding);
 
-            document.ReleaseStrongReference();
-            project.ReleaseStrongReference();
-            workspace.UseReference(static w => w.Dispose());
-
-            compilation.AssertReleased();
-            sol.AssertReleased();
-        }
-
-        [ConditionalFact(typeof(VisualStudioMSBuildInstalled)), Trait(Traits.Feature, Traits.Features.MSBuildWorkspace)]
-        [WorkItem(1088127, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/1088127")]
-        public async Task MSBuildWorkspacePreservesEncoding()
```

Comment:
Where did this go?

Reply:
Line 3047  - Line 3088 from the earlier version of this file seem to be missing.